extension/meminfo.c

```diff
@@ -552,6 +620,23 @@ zend_string * meminfo_escape_for_json(const char *s)
     return s3;
 }
 
+#define MEMORY_LIMIT_ERROR_PREFIX "Allowed memory size of"
```

Comment:
Is there no other way to check for memory exhaustion error?

Reply:
I found this technique in [memprof](https://github.com/arnaud-lb/php-memory-profiler/blob/2b5b133d80608c17891e30df706e1bda826df017/memprof.c#L142) when I was researching how to do it. Unfortunately I didn't find any other options.